reasoner/controller/NestedConjunctionController.java

```diff
@@ -59,7 +59,10 @@ private NestedConjunctionProcessor(Driver<NestedConjunctionProcessor> driver,
 
         @Override
         public void setUp() {
-            setHubReactive(new CompoundStream(this, plan, bounds).buffer());
+            Processor<NestedConjunctionProcessor>.CompoundStream conjunctionStream = new CompoundStream(this, plan, bounds);
+            PoolingStream.BufferedFanStream<ConceptMap> bufferedFanStream = PoolingStream.BufferedFanStream.fanInFanOut(this);
+            conjunctionStream.registerSubscriber(bufferedFanStream);
+            setHubReactive(bufferedFanStream);
```

Comment:
Was an existing bug that could throw an exception when two processors (w/ different bounds) for a conjunction could trigger the negation with the same set of bounds (i.e., When the subset of the conjunction bounds which occur in the negation are identical)

Reply:
Yes, exactly that.